### PR TITLE
Add SQLCommand timeout configuration

### PR DESCRIPTION
**Pull request by jxchinaren**

### CODE REVIEW

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 1]
```diff
@@ -1010,6 +1027,9 @@ private static void printUsage(int exitCode)
         + "  In interactive mode, a value of \"true\" discards any unprocessed input\n"
         + "  and returns to the command prompt. Default: true.\n"
         + "\n"
+        + "[--query-timeout=millisecond_number]\n"
+        + "  Read only queries timeout setting. Default no timeout.\n"
```
Comment (paulmartel) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Add a ":" after Default

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 2]
```diff
@@ -971,6 +987,7 @@ private static void printUsage(int exitCode)
         + "              [--output-format=(fixed|csv|tab)]\n"
         + "              [--output-skip-metadata]\n"
         + "              [--stop-on-error=(true|false)]\n"
+        + "              [--query-timeout=millisecond_number]\n"
```
Comment (paulmartel) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: change millisecond_number to number_of_milliseconds.

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 3]
```diff
@@ -1261,6 +1275,18 @@ else if (arg.startsWith("--ddl-file=")) {
                     printUsage("DDL file not found at path:" + ddlFilePath);
                 }
             }
+            else if (arg.startsWith("--query-timeout=")) {
+                m_hasBatchTimeout = true;
+                m_batchTimeout = Integer.valueOf(extractArgInput(arg));
```
Comment (paulmartel) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Does this give a friendly error message with proper usage advice if you pass "--query-timeout" or "--query-timeout=" or "query-timeout=xin"? Test this.

**Comment by paulmartel** [comment]
This should not get merged without sqlcmdtest support, including negative test cases for bad argument formats.

File: src/frontend/org/voltdb/utils/SQLCommand.java (hunk 3 above)
Comment (jxchinaren) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Exception in thread "main" java.lang.NumberFormatException: For input string: "xin"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:492)
	at java.lang.Integer.valueOf(Integer.java:582)
	at org.voltdb.utils.SQLCommand.main(SQLCommand.java:1281)
xin@volt: bin/sqlcmd --port=xin
Exception in thread "main" java.lang.NumberFormatException: For input string: "xin"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:492)
	at java.lang.Integer.valueOf(Integer.java:582)
	at org.voltdb.utils.SQLCommand.main(SQLCommand.java:1224)

**Comment by jxchinaren** [comment]
Squash merge for it.

**Pull request closed without merging**